package.json

```diff
@@ -31,8 +31,6 @@
     "test:integration": "lerna run test:integration --concurrency 1 --stream",
     "test:without-system-tests":
       "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test",
-    "test:system-tests":
```

Comment:
Do we need to remove this? It seems benign and safer to leave this in and just use the commented out technique up there.

Reply:
I just did exactly what was done in https://github.com/forcedotcom/salesforcedx-vscode/pull/559/files

Reply:
In https://github.com/forcedotcom/salesforcedx-vscode/pull/559/files, I removed this line because I thought that the test script would run all scripts in the package.json that began with "test:*", and I wanted to prevent the system-tests from being run. However, I see that in the .appveyor.yml, line 43 runs `test:without-system-tests` so I'm a bit confused. If the autobuild is already running the script that ignores system tests, then is it the `npm run coverage:system-tests` that invokes the system tests?

Reply:
I'm also not sure if comments in a package.json are a thing. From what I've read there's no comment syntax for the entries that isn't hacky. Removal and re-addition seem like the way to go here.

Reply:
@vazexqi things are ready but I'm not going to submit until you say so.

Reply:
@allileong 
> If the autobuild is already running the script that ignores system tests, then is it the npm run coverage:system-tests that invokes the system tests?

Yes, it is `coverage:system-tests` that invokes the system tests. This can be seen at https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/package.json#L37 where it will invoke the system tests.

@JimSuplizio 
In the interest of time, let's proceed with what we have and restore it back in "W-5507653 - User Story - Re-enable UI system-tests in AppVeyor".